<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-Q4R4JJE" ignored="0" name="Default test" tests="5" failures="1" timestamp="2020-04-04T13:37:47 CEST" time="67.387" errors="0">
  <testcase name="summerDressTest" time="9.371" classname="test.StoreTest"/>
  <testcase name="blueDressTest" time="3.81" classname="test.StoreTest"/>
  <testcase name="registrationTestSingle" time="8.644" classname="test.StoreTest"/>
  <testcase name="registrationTestGroup30" time="35.844" classname="test.StoreTest"/>
  <testcase name="logInTestSingle" time="1.516" classname="test.StoreTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Work] but found [WORK],
	expected [text(),&amp;apos;Tennessee] but found [Tennessee],
	expected [Office] but found [OFFICE],
	expected [contains(text(),&amp;apos;Minnesota] but found [Minnesota],
	expected [Work] but found [WORK],
	expected [//option[contains(text(),&amp;apos;Maine] but found [Maine],
	expected [Home] but found [HOME],
	expected [Maine//option[contains(text(),&amp;apos;Alabama] but found [Alabama],
	expected [Office] but found [OFFICE],
	expected [t(),&amp;apos;Alabama//option[contains(text(),&amp;apos;Texas] but found [Texas]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Work] but found [WORK],
expected [text(),'Tennessee] but found [Tennessee],
expected [Office] but found [OFFICE],
expected [contains(text(),'Minnesota] but found [Minnesota],
expected [Work] but found [WORK],
expected [//option[contains(text(),'Maine] but found [Maine],
expected [Home] but found [HOME],
expected [Maine//option[contains(text(),'Alabama] but found [Alabama],
expected [Office] but found [OFFICE],
expected [t(),'Alabama//option[contains(text(),'Texas] but found [Texas]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at test.StoreTest.logInTestSingle(StoreTest.java:116)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- logInTestSingle -->
</testsuite> <!-- Default test -->
